cmake/configs/default.cmake

```diff
@@ -65,6 +65,7 @@ hunter_config(LLVMCompilerRT VERSION ${HUNTER_Clang_VERSION})
 hunter_config(Leathers VERSION 0.1.6)
 hunter_config(Libcxx VERSION ${HUNTER_Clang_VERSION})
 hunter_config(Libcxxabi VERSION ${HUNTER_Clang_VERSION})
+hunter_config(librtmp VERSION 2.4.0-p0)
```

Comment:
Please remove this one. `ingenue/hunter` is only for `.travis.yml` or `appveyor.yml` updates, see README:
* https://github.com/ingenue/hunter